hadoop-common-project/hadoop-common/src/site/markdown/HadoopDocker.md

```diff
@@ -27,7 +27,7 @@ E.g.
 * [docker-hadoop-3](https://github.com/apache/hadoop/tree/docker-hadoop-3)
 * [docker-hadoop-2](https://github.com/apache/hadoop/tree/docker-hadoop-2)
```

Comment:
But these two are "special branches for running specific hadoop version", sorry if it was unclear.

Check contents of `Dockerfile` on each branch to verify:

https://github.com/apache/hadoop/blob/docker-hadoop-runner-jdk8/Dockerfile

vs.

https://github.com/apache/hadoop/blob/docker-hadoop-3/Dockerfile

Reply:
ooh, I see. I'll update the README

Reply:
I made the changes. Let me know how it looks.

Reply:
Thanks, LGTM.